requirements.txt

```diff
@@ -10,3 +10,4 @@ jupyter==1.0.0
 sphinx-autobuild==2021.3.14
 sphinx-panels==0.5.2
 jinja2==2.11.3
+markupsafe==2.0.1
```

Comment:
@pablo-de-andres I know that you are docs-savvy, so if you have a quick solution for the dependency hell it is welcome. Otherwise this line enough to keep the builds working. Allowing newer versions from everything is a solution, but also requires extra effort fixing other things (relative markdown links stop working for example).